packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.addEventListener.spec.js

```diff
@@ -1,11 +1,81 @@
-it('should accept a listener config as second parameter', () => {
-    const handleEvent = jasmine.createSpy();
+import { createElement } from 'lwc';
+import { extractDataIds } from 'test-utils';
 
-    const elm = document.createElement('div');
-    elm.addEventListener('test', {
-        handleEvent: handleEvent,
+import Container from 'x/container';
+
+function createShadowTree(parentNode) {
+    const elm = createElement('x-container', { is: Container });
+    elm.setAttribute('data-id', 'x-container');
+    parentNode.appendChild(elm);
+    return extractDataIds(elm);
+}
+
+describe('EventTarget.addEventListener', () => {
+    let nodes;
+    beforeEach(() => {
+        nodes = createShadowTree(document.body);
+    });
+
+    it('should accept a function listener as second parameter for all nodes', () => {
+        const targets = [
+            nodes.button,
+            nodes['container_div'],
+            nodes['x-container'].shadowRoot,
+            nodes['x-container'],
+            document.body,
+            document.documentElement,
+            document,
+            window,
+        ];
+
+        const log = [];
+        targets.forEach((node) => {
+            node.addEventListener(
+                'click',
+                function () {
+                    log.push(this);
+                }.bind(node)
+            );
+        });
+
+        nodes.button.click();
+
+        expect(log).toEqual(targets);
+    });
+
+    it('should accept a listener config as second parameter for all nodes except shadow root and host', () => {
```

Comment:
@pmdartus do you want me to remove the test? Or do you think it can be removed when the bug fix goes in?

Reply:
I would suggest reworking the test to add the shadow root and the host. Since they are not passing, I would comment on those 2 use cases for now and add a comment referencing the bug you opened.

This way we have a trace in the code referencing the issue and once this issue is fixed, we will only need to uncomment the test code for those 2 use cases.